cmd/minibroker/main.go

```diff
@@ -87,7 +91,7 @@ func run() error {
 
 func runWithContext(ctx context.Context) error {
 	if flag.Arg(0) == "version" {
-		klog.V(0).Infof("%s/%s", path.Base(os.Args[0]), "0.1.0")
+		fmt.Printf("Minibroker %s - built %s\n", version, buildDate)
```

Comment:
Hmm, how easy would it be to parse out the version? `minibroker version | awk '{ print $2 }'`?

Reply:
We can perhaps use JSON output?

Reply:
Done with https://github.com/kubernetes-sigs/minibroker/pull/84/commits/9235ac94339514d74e668c0739f529d70eb639a0.